src/main/java/org/opensearch/ad/ml/ThresholdingResult.java

```diff
@@ -21,9 +30,51 @@
     private final double grade;
     private final double confidence;
     private final double rcfScore;
+    private long totalUpdates;
+
+    // flag indicating if the anomaly is the start of an anomaly or part of a run of
+    // anomalies
+    private boolean startOfAnomaly;
+
+    // flag indicating if the time stamp is in elevated score region to be
+    // considered as anomaly
+    private boolean inHighScoreRegion;
+
+    /**
+     * position of the anomaly vis a vis the current time (can be -ve) if anomaly is
+     * detected late, which can and should happen sometime; for shingle size 1; this
+     * is always 0
```

Comment:
I'm afraid others who don't read this PR's comments will have difficulty to understand this. How about add some example ?

Reply:
added